python3/src/splib3/interface/serialport.py

```diff
@@ -0,0 +1,43 @@
+import Sofa
+import serial
+import serial.tools.list_ports
+
+
+def getDevicePort(entry, method="manufacturer"):
+    """
+    Returns list of serial ports from chosen method. Available method:
+     - manufacturer
+     - description
+     - serial_number
+    """
+    ports = []
+    if method == "manufacturer":
+        ports = [p for p in serial.tools.list_ports.comports() if entry in p.manufacturer]
+    if method == "description":
+        ports = [p for p in serial.tools.list_ports.comports() if entry in p.description]
+    if method == "serial_number":
+        ports = [p for p in serial.tools.list_ports.comports() if entry in p.serial_number]
+
+    if not ports:
+        Sofa.msg_error("stlib3.interface.serialport",
+                       "No serial port found with " + method + " = " + entry)
+        return
+
+    if len(ports) > 1:
+        Sofa.msg_warning("stlib3.interface.serialport",
+                         "Multiple port found with " + method + " = " + entry + ". Using the first.")
+
+    print("---------------------------------------")
```

Comment:
print  ?

Reply:
I think it's interesting to print the info of the device it found. We can discuss that. 
The print is just to pack the info. I did the same as SofaPython3 does when checking for SOFA_ROOT. 

![image](https://user-images.githubusercontent.com/12150715/224279513-6f2c1f18-c0ca-4b1c-a1aa-897228f5d637.png)

Reply:
I've removed it.